### DISCUSSION TITLE
Rendering with JPG instead of XML

### DISCUSSION THREAD

**Discussion by varadprasad** [mitsuba-renderer/mitsuba3#1385] [Q&A]
How can we use JPG or PNG files instead of XML to create 3D models

**Comment by rtabbara** [comment]
Hi @varadprasad ,

I'm not sure what you're asking is possible. Please have a look at the documentation [here](https://mitsuba.readthedocs.io/en/stable/src/key_topics/scene_format.html) regarding the Mitsuba scene XML format.
